[pysmi] Python 3.13 regression: `tests/test_moduleidentity_smiv2_pysnmp.py::ModuleIdentityTextTestCase::testModuleIdentityDescription`

### Expected behavior

Tests passing in pysmi 1.5.0, like they did in 1.4.4.

### Actual behavior

Tests are failing on Python 3.13.0rc1.

### Detailed steps

```
pip install -e . pysnmp pytest
pytest
```

### Python package information

pysmi 1.5.0, pysnmp 7.0.3, pysnmpcrypto 0.0.4

### Operating system information

Gentoo Linux amd64

### Python information

3.13.0rc1

### (Optional) Contents of your test script

_No response_

### Relevant log output

```shell
========================================================= test session starts =========================================================
platform linux -- Python 3.13.0rc1, pytest-8.3.2, pluggy-1.5.0
rootdir: /tmp/pysmi
configfile: pyproject.toml
collected 262 items                                                                                                                   

tests/test_agentcapabilities_smiv2_pysnmp.py ........                                                                           [  3%]
tests/test_imports_smiv1_pysnmp.py ...........                                                                                  [  7%]
tests/test_imports_smiv2_pysnmp.py ......................                                                                       [ 15%]
tests/test_modulecompliance_smiv2_pysnmp.py ......                                                                              [ 17%]
tests/test_moduleidentity_smiv2_pysnmp.py ...........F.                                                                         [ 22%]
tests/test_notificationgroup_smiv2_pysnmp.py ......                                                                             [ 25%]
tests/test_notificationtype_smiv2_pysnmp.py ......                                                                              [ 27%]
tests/test_objectgroup_smiv2_pysnmp.py .......                                                                                  [ 30%]
tests/test_objectidentity_smiv2_pysnmp.py .......                                                                               [ 32%]
tests/test_objecttype_smiv1_pysnmp.py ..............................                                                            [ 44%]
tests/test_objecttype_smiv2_pysnmp.py ........................................                                                  [ 59%]
tests/test_smiv1_smiv2_pysnmp.py ....                                                                                           [ 61%]
tests/test_traptype_smiv2_pysnmp.py ......                                                                                      [ 63%]
tests/test_typedeclaration_smiv1_pysnmp.py ........................                                                             [ 72%]
tests/test_typedeclaration_smiv2_pysnmp.py ...........................................................                          [ 95%]
tests/test_valuedeclaration_smiv2_pysnmp.py ...........                                                                         [ 99%]
tests/test_zipreader.py ..                                                                                                      [100%]

============================================================== FAILURES ===============================================================
______________________________________ ModuleIdentityTextTestCase.testModuleIdentityDescription _______________________________________

self = <tests.test_moduleidentity_smiv2_pysnmp.ModuleIdentityTextTestCase testMethod=testModuleIdentityDescription>

    def testModuleIdentityDescription(self):
>       self.assertEqual(
            self.ctx["testModule"].getDescription(),
            "\nA\tdescription with\\n\n  various characters: 0~`!@#$%^&*()-_=+[]{}\\|;:'<>,.?/\n"
            + "and a very long line that must not be wrapped despite exceeding the threshold of default word wrap filters.\n\n\\",
            "bad DESCRIPTION",
        )
E       AssertionError: "\nA   description with\\n\n  various charact[149 chars]\n\\" != "\nA\tdescription with\\n\n  various characte[148 chars]\n\\"
E         
E       - A   description with\n
E       ?  ^^^
E       + A	description with\n
E       ?  ^
E           various characters: 0~`!@#$%^&*()-_=+[]{}\|;:'<>,.?/
E         and a very long line that must not be wrapped despite exceeding the threshold of default word wrap filters.
E         
E         \
E        : bad DESCRIPTION

tests/test_moduleidentity_smiv2_pysnmp.py:206: AssertionError
======================================================= short test summary info =======================================================
FAILED tests/test_moduleidentity_smiv2_pysnmp.py::ModuleIdentityTextTestCase::testModuleIdentityDescription - AssertionError: "\nA   description with\\n\n  various charact[149 chars]\n\\" != "\nA\tdescription with\\n\n  various characte[148...
=================================================== 1 failed, 261 passed in 42.41s ====================================================
```

--- a/tests/test_moduleidentity_smiv2_pysnmp.py
+++ b/tests/test_moduleidentity_smiv2_pysnmp.py
@@ -157,7 +157,7 @@ class ModuleIdentityTextTestCase(unittest.TestCase):
     organization"
      CONTACT-INFO "WG-email:\\n   agentx@dorothy.bmc.com\\"
      DESCRIPTION  "
-    A\tdescription with\\n
+    A<TAB>description with\\n
       various characters: 0~`!@#$%^&*()-_=+[]{}\\|;:'<>,.?/
     and a very long line that must not be wrapped despite exceeding the threshold of default word wrap filters.
 
@@ -170,7 +170,9 @@ class ModuleIdentityTextTestCase(unittest.TestCase):
     """
 
     def setUp(self):
-        docstring = textwrap.dedent(self.__class__.__doc__)
+        # As of Python 3.13, tabs are expanded in docstrings, so we have to use
+        # a placeholder and replace it with real tab here.
+        docstring = textwrap.dedent(self.__class__.__doc__.replace("<TAB>", "\t"))
         ast = parserFactory()().parse(docstring)[0]
         mibInfo, symtable = SymtableCodeGen().genCode(ast, {}, genTexts=True)
         self.mibInfo, pycode = PySnmpCodeGen().genCode(

